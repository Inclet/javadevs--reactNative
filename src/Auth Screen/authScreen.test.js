import React from "react";
import { shallow } from "enzyme";
import AuthScreen from "./index";

describe("Auth Screen", () => {
  let wrapper, props;
  beforeEach(() => {
    props = {
      navigation: {
        navigate: jest.fn()
      }
    };
    wrapper = shallow(<AuthScreen {...props} />);
  });
  it("Should match Snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
