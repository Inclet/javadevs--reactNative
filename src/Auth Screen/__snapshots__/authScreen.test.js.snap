// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth Screen RENDERING Should match Snapshot 1`] = `ShallowWrapper {}`;
